packages/rum/src/domain/rumEventsCollection/view/trackViews.ts

```diff
@@ -83,6 +88,11 @@ export function trackViews(location: Location, lifeCycle: LifeCycle) {
   )
 
   return {
+    setCustomTiming(name: string, inInitialView = false, time = performance.now()) {
```

Comment:
A use case where the user wants to record the timing of an event A but only report it if a certain event B happens after. Since we don't offer a way to remove a timing after adding it, the user will have to save the time of event A and only call addTiming when event B happens.

Reply:
do we want to handle this case?

Reply:
The downside is that once it is live we need to maintain it.
Don't hesitate to discuss with @hdelaby about it.

Reply:
After discussion with @hdelaby, we decided to keep it simple for the moment and remove the `time` parameter.